core/state_processor.go

```diff
@@ -623,6 +629,7 @@ func (p *StateProcessor) Process(block *types.WorkObject, batch ethdb.Batch) (ty
 					minGasPrice = new(big.Int).Set(gasPrice)
 				}
 			}
+			calcTxStats(blockMinFee, blockMaxFee, blockAvgFee, fees, numTxsProcessed)
```

Comment:
seems like the function returns values but we dont use it? if that is the case can we remove the returns from the function?